packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -4,7 +4,7 @@
 	}
 	background: $white;
 	border-radius: $radius-block-ui;
-	border: 1px solid $gray-900;
+	border: $border-width solid $gray-900;
 	padding: $grid-unit-15 - 1px; // Subtract the border width.
 	max-height: calc(100vh - 2px); // Subtract the border width (both top and bottom).
```

Comment:
This one would be slightly tricker since we need to output the sass variable. But this should work:

`calc(100vh - #{ $border-width } - #{ $border-width });`